Assets/Scripts/Management/GameManager.cs

```diff
@@ -64,6 +64,8 @@ public class GameManager : MonoBehaviour
             [SerializeField] private LevelManager m_levelTemplate;
             private List<LevelManager> m_levelManagers = new();
             private CardManager m_cardMan;
+            [SerializeField] private MenuPopUp m_pauseMenu;
```

Comment:
Could this not be part of the GameUI script?

Reply:
Fixed